anime_downloader/util.py

```diff
@@ -390,16 +391,20 @@ def get_hcaptcha_cookies(url):
         return pickle.load(open(COOKIE_FILE, 'rb'))
 
 def deobfuscate_packed_js(packedjs):
-    return eval_in_node('eval=console.log; ' + packedjs)
+    return eval_in_node(packedjs)
 
-def eval_in_node(js: str):
+def eval_in_node(js: str, sandbox: str = "{}"):
```

Comment:
Shouldn't this be transparent? We can't assume the next developer (who's going to use `eval_in_node` for implementing a provider) to know how to construct a sandbox string.